### DISCUSSION TITLE
Some changes in the project are not clear

### DISCUSSION THREAD

**Discussion by DimaKozhanovsky** [open-tutorials/cypress#69] [Q&A] [answered]
Colleagues, I'm doing a tutorial "Deploy project Conduit locally" fourth paragraph https://www.epic1h.com/build_conduit.

In VS IDE i have noticed some that some changes in the project were made. 

_Please , help me to understand which changes has benn made and how do they affect the project   ? Thank you in advance_ 🙏

<img width="2089" alt="Screen Shot 2023-02-22 at 22 13 36" src="https://user-images.githubusercontent.com/74324758/220738636-93c5b99d-0ded-4ee5-a054-dbc2b1373b3d.png">

**Comment by breslavsky** [comment, marked as answer]
I see changes in `package-lock.json`. You can read about it more here https://www.youtube.com/watch?v=H3n75nHN5qY
